packages/frontend/src/components/NavBar/index.tsx

```diff
@@ -72,7 +72,7 @@ const NavBar = memo(() => {
                 display="flex"
                 px="md"
                 // FIXME: adjust after removing Blueprint
-                zIndex={10}
+                zIndex={999}
```

Comment:
I've just checked the `zIndex` applied to the `Select` dropdown and it's `300`. Could we go for `500` here? What do you think? 😃

Reply:
I checked it's working fine with 300 as well we can go for 500 but 300 is also that we can stick with.

Reply:
let's go with `300` then pls @jayanand05 👍

Reply:
👍👍

Reply:
I Updated necessary changes request you to look into it?